tests/beaker_tests/cisco_vlan/vlan/vlanlib.rb

```diff
@@ -89,7 +89,7 @@ def self.create_stdvlan_manifest_absent
   # vlan_name, state and shutdown.
```

Comment:
We also need a test case for the default case
  - (example: https://github.com/cisco/cisco-network-puppet-module/blob/develop/tests/beaker_tests/cisco_vlan/vlan/vlanlib.rb#L54)